config.json

```diff
@@ -252,7 +252,15 @@
       "core": false,
       "difficulty": 4,
       "slug": "kindergarten-garden",
-      "topics": null,
+      "topics": [
+          "loops",
+          "arrays",
+          "strings",
+          "logic",
+          "pattern_recognition",
+          "enumerations",
+          "lists"
```

Comment:
Thanks @mhdsmr2, they're now indented evenly :) However, they're still indented four spaces after `topics`, not two spaces after like the other exercises. It would be great if you could change it to be consistent with the rest of the file :)

Reply:
Okay. Really sorry for doing these stupid mistakes. I am just a beginner.

Reply:
No problem at all! Indentation can be difficult to get right and is easy to forget :) Thanks for taking the time to fix these issues :)